common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressTest.java

```diff
@@ -28,96 +16,25 @@
  * </p>
  * @param <TOptions> the options configured for the test.
  */
-public abstract class PerfStressTest<TOptions extends PerfStressOptions> {
-    private final reactor.netty.http.client.HttpClient recordPlaybackHttpClient;
-    private final URI testProxy;
-    private final TestProxyPolicy testProxyPolicy;
-    private String recordingId;
-
-    protected final TOptions options;
-
-    // Derived classes should use the ConfigureClientBuilder() method by default.  If a ClientBuilder does not
-    // follow the standard convention, it can be configured manually using these fields.
-    protected final HttpClient httpClient;
-    protected final Iterable<HttpPipelinePolicy> policies;
-
-    private static final AtomicInteger GLOBAL_PARALLEL_INDEX = new AtomicInteger();
-    protected final int parallelIndex;
-
+public abstract class PerfStressTest<TOptions extends PerfStressOptions> extends ApiPerfTestBase<TOptions> {
```

Comment:
I believe all the `setup` and `cleanup` methods can also be removed, since they are identical to the base class.